Panic is caused when no task found in logs command.

```
ecsher logs task --name exectask-EcsService-ttjK5deaJSV5 --container nginx
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/Mic-U/ecsher/cmd.getTaskLogs()
	/home/runner/work/ecsher/ecsher/cmd/logs.go:134 +0xe9b
github.com/Mic-U/ecsher/cmd.glob..func8(0x1c2a0a0, 0xc00007ae10, 0x1, 0x5)
	/home/runner/work/ecsher/ecsher/cmd/logs.go:51 +0x95
github.com/spf13/cobra.(*Command).execute(0x1c2a0a0, 0xc00007adc0, 0x5, 0x5, 0x1c2a0a0, 0xc00007adc0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1c2a320, 0x101a501, 0x0, 0x0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/Mic-U/ecsher/cmd.Execute()
	/home/runner/work/ecsher/ecsher/cmd/root.go:45 +0x2d
main.main()
	/home/runner/work/ecsher/ecsher/main.go:21 +0x25
```

--- a/cmd/logs.go
+++ b/cmd/logs.go
@@ -82,13 +82,17 @@ func getServiceLogs() {
 	}
 	cluster := config.EcsherConfigManager.GetCluster(describeOptions.Cluster, RootOptions.profile)
 	client := ecs.GetClient(describeOptions.Region, RootOptions.profile)
-	service, err := ecs.DescribeService(client, cluster, []string{logsOptions.Name})
+	services, err := ecs.DescribeService(client, cluster, []string{logsOptions.Name})
 	cobra.CheckErr(err)
+	if len(services) == 0 {
+		fmt.Println("No service found")
+		os.Exit(1)
+	}
 
 	w := new(tabwriter.Writer)
 	w.Init(os.Stdout, 0, 8, 2, ' ', 0)
 	fmt.Fprintln(w, "TIMESTAMP\tID\tMESSAGE")
-	eventLogs := util.AscendingSortServiceLogs(service[0].Events)
+	eventLogs := util.AscendingSortServiceLogs(services[0].Events)
 	for _, eventLog := range eventLogs {
 		fmt.Fprintf(w, "%s\t%s\t%s\n", eventLog.CreatedAt.Local(), *eventLog.Id, *eventLog.Message)
 	}
@@ -129,6 +133,10 @@ func getTaskLogs() {
 	// GetTask
 	tasks, err := ecs.DescribeTask(ecsClient, cluster, []string{logsOptions.Name})
 	cobra.CheckErr(err)
+	if len(tasks) == 0 {
+		fmt.Println("No task found")
+		os.Exit(1)
+	}
 
 	// GetTaskDefinition
 	taskDefinitionArn := tasks[0].TaskDefinitionArn

